lib/bolt/project_migrator.rb

```diff
@@ -33,7 +33,7 @@ def migrate
 
       ok = migrate_inventory && migrate_config
```

Comment:
```suggestion
      ok = migrate_inventory && migrate_config && migrate_modules
```

We should just remove the below `if` block entirely.